store/volume.go

```diff
@@ -4,9 +4,7 @@ package store
 // volumes stored in the store
```

Comment:
IMO, store/volume.go provides the helper function in respect to centralized store where as volume/struct.go has higher abstraction where a volume object is converted into bytes and then subsequent store function is called. This abstraction was also brought in to solve cyclic dependency. as store package was importing volume which is not correct as store should have lower level utility functions.